src/com/sun/jna/Memory.java

```diff
@@ -460,6 +460,20 @@ public void read(long bOff, double[] buf, int index, int length) {
         super.read(bOff, buf, index, length);
     }
 
+    /**
+     * Indirect the native pointer to <code>malloc</code> space, a la
+     * <code>Pointer.read</code>.  But this method performs a bounds checks to
+     * ensure that the indirection does not cause memory outside the
+     * <code>malloc</code>ed space to be accessed.
+     *
```

Comment:
Did you intentionally omit `@param` here since they are the same (other than slight name difference) as `Pointer`?

Reply:
The new override `read(long, Pointer[], int, int)` was missing - it was added because I checked all the `Pointer` returning methods and found it was not present yet. The addition was done so that is is in line with the rest of the overrides.

I checked with NetBeans and it seems, that at least the IDE does the right thing for overriden methods and copies the super class para definitions.